okhttp-tests/src/test/java/com/squareup/okhttp/UrlComponentEncodingTester.java

```diff
@@ -298,7 +298,7 @@ public String encode(int codePoint) {
         return "http://" + value + "@example.com/";
       }
       @Override public String decodedValue(HttpUrl url) {
```

Comment:
"decoded value" calling "encoded username" raises naming flags. Rename these methods too to avoid confusion?